vendor/vendor.json

```diff
@@ -983,14 +983,21 @@
 			"revisionTime": "2016-10-12T14:34:48Z"
 		},
 		{
+			"checksumSHA1": "ITwMGcU7ZNb0nmOHcKXMjK2i69g=",
+			"path": "google.golang.org/api/compute/v0.beta",
+			"revision": "e665075b5ff79143ba49c58fab02df9dc122afd5",
```

Comment:
I could switch it all to v0.beta. From my experience beta api has been quite stable. I can definitely update documentation.